package.json

```diff
@@ -107,5 +115,6 @@
     "**/*.{ts,tsx,js,md,json}": [
       "prettier --write ."
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
any particular reason for introducing this? if so can we separate it into a specific PR. Jamming these things in with feature changes makes creates noise

Reply:
no - will remove. After effect from the yarn berry thing.